homework/03LessonThree.js

```diff
@@ -41,7 +40,7 @@ function returnTheFirstArgument(firstArgument) {
 // What is scope?
 //
 // Your Answer Goes Here:
-//
+// It's right before the function and inside the function.
```

Comment:
I like this definition, but I'd say it's EVERYTHING before the function, not just _right_ inside